revive-lint.toml

```diff
@@ -0,0 +1,25 @@
+ignoreGeneratedHeader = false
+severity = "warning"
+confidence = 0.8
+errorCode = 0
+warningCode = 0
+
+[rule.blank-imports]
+[rule.context-as-argument]
+[rule.context-keys-type]
+[rule.dot-imports]
+[rule.error-return]
+#[rule.error-strings]
```

Comment:
Too bad it doesn't get any more granular, but this is probably nicer than the other way around.

Reply:
yup